pkgs/tools/misc/calamares/0004-Adds-unfree-qml-to-packagechooserq.patch

```diff
@@ -78,7 +78,7 @@ index 000000000..5e36d77d9
 +                    width: 600
 +                    height: 150
 +                    anchors.centerIn: parent
-+                    text: qsTr("NixOS is fully open source, but it also provides software packages with unfree licenses. By default unfree packages are not allowed, but you can enable it here. If you check this box, software installed might have additional End User License Agreements (EULAs) attached. If not enabled, some hardware might not work fully when no suitable open source drivers are available.<br/>")
++                    text: qsTr("NixOS is fully open source, but it also provides optional software packages that do not respect users' freedom to run, copy, distribute, study, change and improve the software, and are commonly not open source. By default such "unfree" packages are not allowed, but you can enable it here. If you check this box, you agree that unfree software may be installed which might have additional End User License Agreements (EULAs) that you need to agree to. If not enabled, some hardware (notably Nvidia GPUs and some WiFi chips) might not work or not work optimally.<br/>")
```

Comment:
As mentioned in the above thread, at the moment proprietary nvidia drivers are not installed by the graphical installer. Only whether to enable unfree packages and force disable broadcom stuff.

Reply:
Since this is a setting that does not only affect the bootloader, I think it's fine to also mention the Nvidia driver which would require this if the user were to activate it, even if the installer doesn't install it on its own.